frontends/ol-components/src/components/Button/Button.tsx

```diff
@@ -188,6 +182,19 @@ const buildStyles = (props: ButtonStyleProps & { theme: Theme }) => {
         color: colors.silverGray,
       },
     },
+    variant === "bordered" && {
+      backgroundColor: colors.white,
+      color: colors.silverGrayDark,
+      border: `1px solid ${colors.silverGrayLight}`,
+      ":hover:not(:disabled)": {
+        backgroundColor: colors.lightGray1,
+      },
+      ":disabled": {
+        backgroundColor: colors.lightGray2,
+        border: `1px solid ${colors.silverGrayDark}`,
+        color: colors.silverGrayDark,
+      },
```

Comment:
The color darkens on hover, and the border when disabled should be same color as background. (You're right that Figma says the border color is silverGrayDark, but figma also says the border width is 0. I think setting the width to zero will mess up the height. I don't trust figma css :p) 
```suggestion
      ":hover:not(:disabled)": {
        backgroundColor: colors.lightGray1,
        color: colors.darkGray2,
      },
      ":disabled": {
        backgroundColor: colors.lightGray2,
        borderColor: colors.lightGray2
        color: colors.silverGrayDark,
      },
```